File: src/parse2.py
#!/usr/bin/python
import re

lines = open('redis.properties', 'r').readlines()
config = {k: v for k, v in
           [l.split('=')
             for l in
                 map(lambda s: s.strip(), lines)
             if not(re.match('^ *($|#)', l))]}

print config

File: src/parse2.js
var fs = require('fs');

lines = fs.readFileSync('redis.properties').toString().split('\n')

config = lines.filter(s => !/^ *($|#)/.test(s))
              .map(s => s.split('='))
              .reduce((o,[k,v]) => {o[k] = v; return o}, {})

console.log(config)

